src/components/BookCard/BookCardBody.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import BookPublishDate from '../BookDetails/BookPublishDate';
+import BookQuote from '../BookDetails/BookQuote';
+import BookAuthor from '/src/components/BookDetails/BookAuthor.jsx';
+import BookTitle from '/src/components/BookDetails/BookTitle.jsx';
+
+export default function BookCardBody({
+	title,
+	publishDate,
+	quote,
+	author,
+}) {
+	return (
+		<div className="card-body">
```

Comment:
Same comment as the other PR, but I would expect all the HTML and CSS for this component to exist in one component (BookCard.jsx) so that it's easy to read and make changes.